finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Result.scala

```diff
@@ -207,6 +223,9 @@ case class Field(
 ) extends Result {
   def id: String = if (name.isEmpty) origName else name
   override val toString = "Field(%s)".format(id)
+
+  def isUnsigned(): Boolean = (flags & FieldAttributes.UnsignedBitMask) > 0
```

Comment:
Some style nits for ya that I missed earlier:

We typically leave parens off of these types of methods (both defining and calling) unless they mutate in some way. Since this doesn't, please omit here and in isSigned